modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm

```diff
@@ -25,6 +25,10 @@
 
 	family_heirlooms = list(/obj/item/clothing/neck/stethoscope, /obj/item/roller, /obj/item/book/manual/wiki/security_space_law)
 
+/datum/job/security_medic/after_spawn(mob/living/carbon/human/H, mob/M)
```

Comment:
what is this H M bullshit bro u should know better than that

Reply:
i copied how gandalf did it bro....

Reply:
newbie error